source/js/l10n.js

```diff
@@ -0,0 +1,38 @@
+import "core-js/stable";
+import "regenerator-runtime/runtime";
+import "intl-pluralrules";
+import { FluentBundle, FluentResource } from "@fluent/bundle/compat";
+import { negotiateLanguages } from "@fluent/langneg";
+import { getCurrentLanguage } from "./components/petition/locales";
+
+import deMessages from "../../network-api/locale/de/messages.ftl";
+import esMessages from "../../network-api/locale/es/messages.ftl";
+import frMessages from "../../network-api/locale/fr/messages.ftl";
+import plMessages from "../../network-api/locale/pl/messages.ftl";
+import ptMessages from "../../network-api/locale/pt_BR/messages.ftl";
+import sourceMessages from "../../network-api/locale/templates/messages.ftl";
```

Comment:
So this code includes all translations in the final bundle, right? This might be fine for now since there aren't many locales nor messages to translate, but it might be a good idea to have a plan for when bundling all translations will increase the bundle size by too much. Then, you'll likely want to `fetch` the FTL files just for the current locale and for English (for fallback) on the runtime.

Reply:
We’re importing files for all locales, but only creating bundles for English and the current locale. Since the import is done at build time, we don’t know yet what the page locale will be.
`fetch` seem indeed to do what I was looking for. Let’s see if I manage to get something working :)

Reply:
Right, sorry, I wasn't clean with my usage of the word `bundle`. In my previous comment I meant `bundle` as in `the JS file that has all your code and is sent to the users`, not the `FluentBundle` :)